src/Middleware/RateLimiting/src/PublicAPI.Unshipped.txt

```diff
@@ -0,0 +1,11 @@
+Microsoft.AspNetCore.RateLimiting.RateLimitingExtensions
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.Limiter.get -> System.Threading.RateLimiting.PartitionedRateLimiter<Microsoft.AspNetCore.Http.HttpContext!>!
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.Limiter.set -> void
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.OnRejected.get -> System.Func<Microsoft.AspNetCore.Http.HttpContext!, System.Threading.RateLimiting.RateLimitLease!, System.Threading.Tasks.Task!>!
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.OnRejected.set -> void
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.RateLimitingOptions() -> void
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.RejectionStatusCode.get -> System.Net.HttpStatusCode
+Microsoft.AspNetCore.RateLimiting.RateLimitingOptions.RejectionStatusCode.set -> void
+static Microsoft.AspNetCore.RateLimiting.RateLimitingExtensions.UseRateLimiting(this Microsoft.AspNetCore.Builder.IApplicationBuilder! app) -> Microsoft.AspNetCore.Builder.IApplicationBuilder!
+static Microsoft.AspNetCore.RateLimiting.RateLimitingExtensions.UseRateLimiting(this Microsoft.AspNetCore.Builder.IApplicationBuilder! app, Microsoft.AspNetCore.RateLimiting.RateLimitingOptions! options) -> Microsoft.AspNetCore.Builder.IApplicationBuilder!
```

Comment:
Can you add all of this in normal C# syntax to #37384 and add the `api-ready-for-review` label so we discuss this Monday?

Reply:
Done